dataprofiler/profilers/unstructured_text_profile.py

```diff
@@ -182,7 +190,20 @@ def __add__(self, other):
                           "since there were conflicting values for case "
                           "sensitivity between the two profiles being merged.")
 
-        
+        if self._num_most_common_words and other._num_most_common_words:
+            merged_profile._num_most_common_words = max(
+                self._num_most_common_words, other._num_most_common_words)
+        else:
+            merged_profile._num_most_common_words = \
+                self._num_most_common_words or other._num_most_common_words
```

Comment:
If either is `None` isn't the max then `None? Hence' maybe it should be:
```python
merged_profile._top_k_words = None
if self._top_k_words and other._top_k_words:
    merged_profile._top_k_words = max(self._top_k_words, other._top_k_words)
```

same for below with chars. But do consider ordering discussion as above

Reply:
currently, if one is None, I get the value from the other. But we can discuss about the expected result on this.

Reply:
following the logic of None - max, changed it now